helper/hashitools/project_test.go

```diff
@@ -21,6 +22,41 @@ func TestProjectLatestVersion(t *testing.T) {
 	}
 }
 
+type stubInstaller struct {
+	path string
+}
+
+func (t *stubInstaller) InstallAsk(installed, required, latest *version.Version) (bool, error) {
+	return false, nil
+}
+func (t *stubInstaller) Install(*version.Version) error { return nil }
+func (t *stubInstaller) Path() string                   { return t.path }
+
+// https://github.com/hashicorp/otto/issues/70
+func TestVersion_vagrantStdErrWarning(t *testing.T) {
+	path, err := filepath.Abs(filepath.Join(
```

Comment:
Is this because of the `./` below? Is there a simple way to just make it work properly instead?

Maybe `filepath.Abs(filepath.Join(".", "test-fixtures", ...)` ?